src/common/buffer_seastar.cc

```diff
@@ -20,6 +20,7 @@
 using temporary_buffer = seastar::temporary_buffer<char>;
 
 namespace ceph::buffer {
+inline namespace v14_2_0 {
```

Comment:
@dillaman  probably we don't need to expose `raw_seastar_foreign_ptr` to the public ? as `raw_seastar_foreign_ptr` is only used by crimson project at this moment.

Reply:
I only did that since ```create_foreign``` was in the public API. I can wrap the ```createXYZ``` methods in the namespace and keep the classes private.

Reply:
thanks. that'd be better. actually, i didn't realize that `create_foreign` was exposed! it's used to convert a seastar `seastar::temporay_buffer` to `ceph::buffer::raw`.